pydata_sphinx_theme/__init__.py

```diff
@@ -18,6 +18,17 @@
 
 logger = logging.getLogger(__name__)
 
+# by default, use the stdlib html parser
+BS4_PARSER = "html.parser"
+
+try:
+    # but prefer lxml
+    import lxml
+    BS4_PARSER = "lxml"
+    logger.info("Using lxml for HTML parsing")
+except ImportError:
```

Comment:
might need this too?

```suggestion
except (ImportError, ModuleNotFoundError):
```

Reply:
nawp, think we're good:

```
ModuleNotFoundError.__mro__
(ModuleNotFoundError, ImportError, Exception, BaseException, object)
```